apps/src/music/player/MusicPlayer.ts

```diff
@@ -455,7 +456,9 @@ export default class MusicPlayer {
       return null;
     }
 
-    const folder: SoundFolder | null = library.getFolderForFolderId(instrument);
+    const folder: SoundFolderInstrument | null = library.getFolderForFolderId(
+      instrument
+    ) as SoundFolderInstrument;
```

Comment:
Similarly here, could use a type check to validate the returned folder rather than casting, or alternatively a separate specific `getInstrumentFolderForId(): SoundFolderInstrument` function